src/app/modules/login/login.component.ts

```diff
@@ -1,12 +1,33 @@
 import { Component, OnInit } from '@angular/core';
+import { ApiService } from '../../../backend-client';
```

Comment:
We also need [authGuard](https://jasonwatmore.com/post/2019/06/22/angular-8-jwt-authentication-example-tutorial#auth-guard-ts) try implement by following this example